src/index.js

```diff
@@ -0,0 +1,157 @@
+const city = document.getElementById('city_name')
+let inputCity = document.getElementById('city_input')
+
+function changeCity(event) {
+    if (event.key === 'Enter') {
+        city.textContent = inputCity.value;
+    }
+}
+inputCity.addEventListener('keydown', changeCity)
+inputCity.addEventListener('keydown', updateWeather)
+
+
+
+const increaseTempButton = document.getElementById('increase_temp')
+const decreaseTempButton = document.getElementById('decrease_temp')
+
+
+function increaseTemp() {
+    const temperature = document.getElementById('temp_no');
+    let current_temp = Number(temperature.textContent)
+
+    temperature.textContent = current_temp + 1;
+    tempColor(current_temp + 1);
+    changeLandscape(current_temp + 1)
+}
+
+increaseTempButton.addEventListener('click', increaseTemp)
+
+function decreaseTemp() {       
+    const temperature = document.getElementById('temp_no');
+    let current_temp = Number(temperature.textContent)
+
+    temperature.textContent = current_temp - 1;
+    tempColor(current_temp - 1);
+    changeLandscape(current_temp + 1)
+}
+decreaseTempButton.addEventListener('click', decreaseTemp)
+
+function changeLandscape(temp) {
+    let landscape = document.getElementById('fashion_landscape');
+    if (temp < 60) {
+        landscape.src = "assets/cold_outfit.jpeg"
+    } else if (temp >= 60 && temp < 70) {
+        landscape.src = "assets/60_degree_outfit.jpeg"
+    } else if (temp >= 70 && temp < 80) {
+        landscape.src = "assets/cardigan_and_jeans.jpeg"
+    } else {
+        landscape.src = "assets/80_degree_outfit.jpeg"
+    }
+
+}
+
+function tempColor(temp) {
+    let temperature = document.getElementById('temp');
+    if (temp <= 49) {
+        temperature.className = 'freezing';
+    } else if (temp > 49 && temp < 60) {
+        temperature.className = 'cold';
+    } else if (temp >= 60 && temp < 70) {
+        temperature.className = 'chilly';
+    } else if (temp >= 70 && temp < 80) {
+        temperature.className = 'hot';
+    } else {
+        temperature.className = 'scorching';
+    }
+}
+
+let skySelector = document.getElementById('sky_selector');
+let skyImage = document.getElementById('pinterest-sky');
```

Comment:
There are a lot of variable declarations scattered around this file so I'd recommend finding a way to organize these a little better:
* Declaring variables at the smallest scope necessary, so initializing variables like `skyImage` inside of the function they're used in instead of at the global scope
* Declaring variables that are globally scoped because they're used in more than one place at the top of the file
* Declaring variables that are globally scoped but only used once directly above the line of code that uses the variable

I'd also recommend grouping the event handler functions in one section and then having the event listener registrations in another section.